native/machine-coding/atm-simulator/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="../../logo.svg" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="script.js" type="module"></script>
+    <script src="../../helpers/navbar.js" type="module"></script>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container">
+        <h1>ATM Simulator</h1>
+        <label for="pin">Enter PIN:</label>
+        <input type="password" id="pin" placeholder="Enter your PIN">
+        <button onclick="authenticate()">Submit</button>
+
+        <div class="options">
+            <button onclick="checkBalance()">Check Balance</button>
+            <button onclick="deposit()">Deposit</button>
+            <button onclick="withdraw()">Withdraw</button>
+            <button onclick="logout()">Logout</button>
```

Comment:
Use `click` **event listeners** in your script file instead of using `onclick` in HTML